src/pages/profile-page/profile-page.tsx

```diff
@@ -7,17 +7,8 @@ import styles from './profile-page.module.css';
 import { GeneralInfo } from './components/general-info';
 import { ChangePassword } from './components/change-password';
 import { Addresses } from './components/addresses';
-import { PagePath } from '../../router/enums';
-import { useEffect } from 'react';
-import { useNavigate } from 'react-router';
 
 export const ProfilePage = () => {
-  const navigate = useNavigate();
-  useEffect(() => {
-    if (!userStore.isAuth) {
-      void navigate(PagePath.loginPage);
-    }
-  }, [userStore.isAuth, navigate]);
```

Comment:
ну или так
```suggestion
  const navigate = useNavigate();
  useEffect(() => {
    if (!userStore.isAuth && !userStore.isInitLoading) {
      void navigate(PagePath.loginPage);
    }
  }, [userStore.isAuth, navigate]);
```
только дефолтный isInitLoading в тру перевести
но наверное мало кто полезет проверять насколько этот роут приватный, так что пока можно и так